src/button/component.jsx

```diff
@@ -355,30 +355,6 @@ export const Button : Component<ButtonOptions> = create({
             }
         },
 
-        localhostUrl: {
```

Comment:
I think you can delete these props from the Types as well: https://github.com/paypal/paypal-checkout-components/blob/v4/src/button/component.jsx#L108-L109